core/lib/libtomcrypt/rsa.c

```diff
@@ -15,6 +15,7 @@
 #include "acipher_helpers.h"
 
 
+#if !defined(CFG_CORE_SE05X)
```

Comment:
I'd rather add `__maybe_unused` instead. The function will be discarded by the linker anyways (GC phase), and it's good to avoid ifdefs and increase compile coverage.

Reply:
@cneveux glad you're chiming in :) I totally agree, it's just that I am not yet very familiar with the crypto driver API.